sbnanaobj/StandardRecord/SRNuID.cxx

```diff
@@ -0,0 +1,30 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRNuID.cxx
+// \brief   SRNuID object for Neutrino ID score features (MVA inputs). This SR code copied/based on other SR objects.
+// \author  howard@fnal.gov
+////////////////////////////////////////////////////////////////////////
+
+#include "sbnanaobj/StandardRecord/SRNuID.h"
+
+namespace caf
+{
+  SRNuID::SRNuID()
+  {  }
+
```

Comment:
Given its emptiness, the constructor can be removed, unless there is the explicit intention of forbidding [aggregate initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization) (which I would not forbid).

Reply:
Sorry I missed your comments before finalizing my statement below. There is no intent from me here, but other Standard Record objects have empty constructors apart from variable assignment. Does this carry the same impact w.r.t. aggregate initialization? If it _does_ then I defer to @cjbackhouse and/or @FernandaPsihas to add their opinion. If not, then I guess these could be removed.

Reply:
It does.

Reply:
Yeah, no opinion either way about empty constructors.